benches/bloom_bench.rs

```diff
@@ -0,0 +1,64 @@
+use bytes::{BufMut, BytesMut};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use slatedb::filter::{filter_hash, BloomFilterBuilder};
+fn bloom_filter_insert_benchmark(c: &mut Criterion) {
+    let mut builder = BloomFilterBuilder::new(10); // Set bits_per_key
+    let key_sz = std::mem::size_of::<u32>();
+
+    c.bench_function("bloom_filter insert 10000000 keys", |b| {
+        b.iter(|| {
+            for i in 0..10000000 {
+                let mut bytes = BytesMut::with_capacity(key_sz);
+                bytes.put_u32(i);
+                builder.add_key(black_box(bytes.freeze().as_ref())); // Use `black_box` to prevent compiler optimizations
+            }
+        })
+    });
+}
+fn bloom_filter_build_benchmark(c: &mut Criterion) {
```

Comment:
rather than make all the bloom filter types public, could we just add a public filter_bench module to the lib and put these methods in there? They can/should still be called from `bloom_bench.rs`, since it looks like Criterion can't be embedded alongside the source code.